tests/components/navbar/navbar.test.jsx

```diff
@@ -55,7 +57,10 @@ describe('components/navbar/Navbar', () => {
     test('should match snapshot, if not licensed', () => {
         global.window.mm_license.IsLicensed = 'false';
```

Comment:
I think this line (and similar ones in the following tests) are no longer needed.

Reply:
Unfortunately, there's some remaining cleanup for higher level components that makes these global assignment still necessary -- for now. Rest assured, it's all going away soon :)